lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -242,6 +244,79 @@ class DetailsRenderer {
     return tableElem;
   }
 
+  /**
```

Comment:
this is called `_renderOpportunityTable` for now, but I'd like to migrate all audits with table details to this one and eventually just have it be `_renderTable`. The changes from the original mean we only have so many types of things that can fit in the table cells, so just call them out explicitly instead of having nested details that can themselves contain more details, etc.

E.g. here we get rid of the "URLs" that are actually `{type: 'code', value: txt}`, and instead just say if it doesn't look like a URL, render it in `<pre></pre>` tags

Reply:
Does this mean we'd eliminate the `fallbackType/fallbackKey` approach? Also this method feels a bit long, want to break it up? :)

Reply:
> I like this. Does this mean we'd eliminate the fallbackType/fallbackKey approach?

I guess it comes down to how flexible we need it to end up being. It doesn't seem like we have a ton of cases where we need fallbacks, so we can probably get by with using hybrid `valueType`s, like `url` is here (maybe we'd want to rename them based on their actual combined type? e.g. `'urlOrCode'`, but something better :)

Reply:
can you add a comment with this todo?